src/test/java/com/uid2/shared/attest/UidCoreClientTest.java

```diff
@@ -59,6 +55,37 @@ public void Download_Succeed_RequestSentWithExpectedParameters() throws IOExcept
         when(mockHttpClient.get("https://download", expectedHeaders)).thenReturn(mockHttpResponse);
 
         uidCoreClient.download("https://download");
+        verify(mockAttestationTokenRetriever, times(1)).attest();
```

Comment:
Can you wrap the `verify()`s under an `assertAll()`? Similar comment for the other test